e2e/cypress/integration/ui/settings/02_team_mgmt.spec.ts

```diff
@@ -1,25 +1,47 @@
 describe('team management', () => {
+  let adminToken = '';
+  const now = Cypress.moment().format('MMDDYYhhmm');
+  const cypressPrefix = 'cypress-test';
+  const teamName = `${cypressPrefix} team ${now}`;
+  const customTeamID = `${cypressPrefix}-testing-team-custom-id-${now}`;
+  const project1ID = `${cypressPrefix}-project1-${now}`;
+  const project1Name = `${cypressPrefix} project1 ${now}`;
+  const project2ID = `${cypressPrefix}-project2-${now}`;
+  const project2Name = `${cypressPrefix} project2 ${now}`;
+  const generatedTeamID = teamName.split(' ').join('-');
+  const unassigned = '(unassigned)';
+
+  let iamVersion = <string><Object>Cypress.env('IAM_VERSION');
+  // assume 2.1 if not in CI. if you wish something different start cypress with
+  // IAM_VERSION set to what you are testing.
+  if (iamVersion === undefined) {
+    iamVersion = 'v2.1';
```

Comment:
I'm not sure we should assume `v2.1` if run locally. I've been trying to figure out a way to have the test check for us if the environment variable isn't set (unfortunately not having much luck). So in the meantime it probably makes more sense to set this to `v2`, since that's most likely what a local environment will be set to

Reply:
Fine by me! Will update.